great_expectations_cloud/logging/logging_cfg.py

```diff
@@ -12,11 +12,13 @@
 
 from typing_extensions import override
 
+from great_expectations_cloud.agent.config import GxAgentEnvVars
+
 LOGGER = logging.getLogger(__name__)
 
 DEFAULT_LOG_FILE: Final[str] = "logfile"
 DEFAULT_LOG_DIR = "logs"
-SERVICE_NAME: Final[str] = "gx-agent"
+SERVICE_NAME: Final[str] = GxAgentEnvVars().service_name
```

Comment:
I would flip this dependency if possible.
Make the default `GxAgentEnvVars.service_name` come from a constant, rather than this constant coming from `GxAgentEnvVars().service_name`.

Reply:
Thanks - I added the default value to the existing `great_expectations_cloud/agent/constants.py ` file.

Because I need the runner to be able to override this value, I still need to keep it as part of `GxAgentEnvVars` and then use the env vars to get the SERVICE_NAME in this file. I don't know exactly how it works, but the SERVICE_NAME is added to the logger log event. Here's an example log pulled from ArgoCD:
```
{
  "args": [],
  "levelname": "INFO",
  "filename": "agent.py",
  "module": "agent",
  "exc_info": null,
  "stack_info": null,
  "funcName": "_handle_event_as_thread_exit",
  "created": 1720803645.03965,
  "relativeCreated": 154868265.17105103,
  "event_type": "run_scheduled_checkpoint.received",
  "correlation_id": "a01d7125-a5b7-4791-8e76-198a01958551",
  "event": "Completed job",
  "level": "INFO",
  "logger": "great_expectations_cloud.agent.agent",
  "timestamp": "2024-07-12T17:00:45.039650+00:00",
  "environment": "production",
  "service_name": "gx-agent"
}
```

Reply:
This sounds like a symptom of using a module level variable.

Where does this `logging_cfg.SERVICE_NAME` variable actually get used?

I would not define a module level variable here at all if possible and instead pull the `GxAgentEnvVars().service_name` when its actually needed.